OpenSSL/SSL.py

```diff
@@ -318,6 +319,56 @@ def wrapper(ssl, out, outlen, in_, inlen, arg):
         )
 
 
+class _AlpnSelectHelper(_CallbackExceptionHelper):
+    """
+    Wrap a callback such that it can be used as an ALPN selection callback.
+    """
+    def __init__(self, callback):
+        _CallbackExceptionHelper.__init__(self)
+
+        @wraps(callback)
+        def wrapper(ssl, out, outlen, in_, inlen, arg):
+            try:
+                conn = Connection._reverse_mapping[ssl]
+
+                # The string passed to us is made up of multiple
+                # length-prefixed bytestrings. We need to split that into a
+                # list.
+                instr = _ffi.buffer(in_, inlen)[:]
+                protolist = []
+                while instr:
+                    l = indexbytes(instr, 0)
+                    proto = instr[1:l+1]
```

Comment:
Spaces around operators - `l + 1`.  Also maybe a variable name that's not lower case latin letter L (it's a particularly unpleasant single-letter variable).